src/Discord.Net.Core/Entities/Permissions/GuildPermissions.cs

```diff
@@ -70,7 +70,9 @@ public struct GuildPermissions
         /// <summary> If <c>true</c>, a user may use voice-activity-detection rather than push-to-talk. </summary>
         public bool UseVAD => Permissions.GetValue(RawValue, GuildPermission.UseVAD);
         /// <summary> If True, a user may use priority speaker in a voice channel. </summary>
-        public bool PrioritySpeaker => Permissions.GetValue(RawValue, ChannelPermission.PrioritySpeaker);
+        public bool PrioritySpeaker => Permissions.GetValue(RawValue, GuildPermission.PrioritySpeaker);
```

Comment:
Oh this is probably worth highlighting. Thoughts?

Reply:
Values are the same, but yeah that's a good catch.